src/components/Lists/ListBase/ListBase.stories.tsx

```diff
@@ -30,22 +42,50 @@ const simpleListItems = [
 ];
 
 export const SimpleList: Story<ListBaseProps> = (args) => (
-    <ListBase {...args}>
-        <ListItemBase>
-            <ListItemIconBase color="secondary">
-                <InboxIcon />
-            </ListItemIconBase>
-            <ListItemTextBase primary="Inbox" color="secondary-main" />
-        </ListItemBase>
-
-        {simpleListItems.map((item, index) => (
-            <ListItemBase key={index}>
-                <ListItemIconBase>{item.icon}</ListItemIconBase>
-                <ListItemTextBase primary={item.text} />
+    <BoxBase width={360}>
+        <ListBase {...args}>
+            <ListItemBase button>
+                <ListItemIconBase bgcolor="primary" color="secondary">
+                    <InboxIcon />
+                </ListItemIconBase>
+                <ListItemTextBase primary="Inbox" color="secondary-main" />

```

Comment:
I think you should have a ListItemNavigation that wraps this block so that the color is decided by it's variant: highlighted / default